frontend/src/hooks/useApi.ts

```diff
@@ -176,6 +179,60 @@ export const useApi = (): PimsAPI => {
     [],
   );
 
+  /**
+   * Make an AJAX request to update the specified building financials.
+   * @param id The building primary key 'id' value.
+   * @param building - the building data to be update
+   * @returns A promise containing the building.
+   */
+  axios.updateBuilding = useCallback(
+    async (id: number, building: IApiProperty) => {
+      const { data } = await axios.put<IBuilding>(
+        `${ENVIRONMENT.apiUrl}/properties/buildings/${id}/financials`,
+        building,
+      );
+      return data;
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [],
+  );
+
+  /**
+   * Make an AJAX request to update the specified parcel financials.
+   * @param id The parcel primary key 'id' value.
+   * @param parcel - the parcel data to be update
+   * @returns A promise containing the parcel.
+   */
+  axios.updateParcel = useCallback(
+    async (id: number, parcel: IApiProperty) => {
+      const { data } = await axios.put<IParcel>(
+        `${ENVIRONMENT.apiUrl}/properties/parcels/${id}/financials`,
+        parcel,
+      );
+      return data;
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [],
+  );
+
+  /**
```

Comment:
is this a duplicate of the other updateBuilding function?

Reply:
Resolved